crates/llama-cpp-bindings/Cargo.toml

```diff
@@ -4,9 +4,18 @@ version = "0.10.0-dev.0"
 edition = "2021"
 
 [features]
+default = ["native", "avx", "fma"]
```

Comment:
```suggestion
# non-windows
default = ["avx", "avx2", "fma", "f16c"]

# windows
default = ["avx", "avx2", "fma"]
```

This should be the actual default tabby is using, could you update?

Reply:
Ok. What should I do with avx2? I disabled it by default in order to fix https://github.com/TabbyML/tabby/issues/1142. Was inspired by https://github.com/ollama/ollama/issues/644#issuecomment-1783903043

Reply:
Done with avx2 on by default